app/controllers/application_controller.rb

```diff
@@ -7,6 +7,7 @@ class ApplicationController < ActionController::Base
   before_action :is_logged_in
   before_action :setup_query
   before_action :refresh_urs_if_needed, except: [:logout, :refresh_token]
+  before_filter :groups_enabled?
```

Comment:
I was mistaken about the provider holdings parent - I thought that the logged out state also used the ProviderHoldingsController, but actually it is using WelcomeController.